packages/apps/docs/src/scripts/importChangelogs/index.ts

```diff
@@ -175,7 +175,9 @@ export const importChangelogs = async (): Promise<IScriptResult> => {
 
   deleteTempDir();
 
-  process.exitCode = 1;
+  if (errors.length > 0) process.exitCode = 1;
```

Comment:
🥲

Reply:
I'm not sure if there is a better way to do this..